frontend/public/css/style.css

```diff
@@ -141,11 +141,15 @@ input[type=range]:focus {
 
 /* modals */
 .modal img {
-    position: relative;
-	left: 500px;
-	bottom: 100px;
-    height: 250px;
-    width: 350px;
+    width: 100%;
+}
+.bootstrap-tagsinput {
+	position: absolute;
```

Comment:
Yes, I would appreciate it, if you can fix that when you are working on the styling part.